tests/unittests/config/test_cc_ca_certs.py

```diff
@@ -122,10 +127,10 @@ def test_single_trusted(self):
 
     def test_multiple_trusted(self):
         """Test that multiple certs get passed to add_ca_certs."""
-        config = {"ca-certs": {"trusted": ["CERT1", "CERT2"]}}
+        config = {"ca_certs": {"trusted": ["CERT1", "CERT2"]}}
 
         for distro_name in cc_ca_certs.distros:
-            self._mock_init()
+            self._mock_init(distro_name)
             cloud = get_cloud(distro_name)
```

Comment:
and here,

Reply:
again, this is after _mock_init is called where the mock is defined.